nevergrad/optimization/recastlib.py

```diff
@@ -172,7 +193,8 @@ class NonObjectOptimizer(base.ConfiguredOptimizer):
         - SQP (or SLSQP): very powerful e.g. in continuous noisy optimization. It is based on
           approximating the objective function by quadratic models.
         - Powell
-        - NLOPT (https://nlopt.readthedocs.io/en/latest/; uses Sbplx, based on Subplex)
+        - NLOPT* (https://nlopt.readthedocs.io/en/latest/; by default, uses Sbplx, based on Subplex);
+            can be NLOPT, NLOPT1, NLOPT2, ..., NLOPT13.
```

Comment:
even for a hack I don't quite like it, the numbers do not make any sense.
`NLOPT_LN_SBPLX` instead of `NLOPT1` would be better for instance, it's easy to extract automatically the names that work, and if it does not match we should return an explicit exception with appropriate options